### DISCUSSION TITLE
PDF viewer in the same way as for Image

### DISCUSSION THREAD

**Discussion by mateustavaresps** [ant-design/ant-design#41149] [Q&A]
I want to see a pdf the same as you have for the image

![image](https://user-images.githubusercontent.com/126293948/224061778-cb078974-4e74-40f5-ad2b-7236f3e5f5b4.png)

Does anyone have an idea to do this?

**Comment by zombieJ** [comment]
Since antd do not support PDF view. It can only to customize component to imitate Image viewer design.

**Comment by dipakpatil5050** [comment]
can you support for PDF Viwer in react app ?
